### DISCUSSION TITLE
Tile multiple windows side-by-side with one command?

### DISCUSSION THREAD

**Discussion by quinncomendant** [rxhanson/RectanglePro-Community#394] [Q&A] [answered]
I just purchase Rectangle Pro and am excitedly exploring its features. One of the reasons I upgraded from the free version of Rectangle is for the “Multiple Windows” commands. I need a way to tile windows full-height, side-by-side, in `1x2`, `1x3`, and `1x4` configurations. How do I do this? 

Each global tiling command should also have an app-only tiling version. The `2x2 Tiles` and `2x3 Tiles` commands apply to all windows, but there is no equivalent to tile only windows for the foremost app. @rxhanson would you be willing to add these options? Thanks!

**Comment by rxhanson** [comment, marked as answer]
Thanks for purchasing, and for the feedback. Although not precisely what you're looking for, here is how to make a 2x2 grid in App Layouts:

![image](https://github.com/rxhanson/RectanglePro-Community/assets/13651296/ec09e83e-8fab-4f16-9764-a7fe936c07ae)

I'll put the frontmost app idea on my backlog, and consider more default tiling options as well.
